### PR TITLE
Templatization refactor

### PR DESCRIPTION
**Pull request by Folrig**
*Issue #, if available:*

*Description of changes:*
Refactor to include prescriptive directions and additional templates for more scalable consumption

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: content/1_ModuleOne/11_Foreward.ee.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
```
Comment (eugenemu) [inline content/1_ModuleOne/11_Foreward.ee.md]: We need to remove the .ee.md - but this will be useful for a separate branch that we have.

File: content/1_ModuleOne/11_Foreward.ee.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+title: "Foreward" # MODIFY THIS TITLE IF APPLICABLE
```
Comment (eugenemu) [inline content/1_ModuleOne/11_Foreward.ee.md]: Foreword*

File: README.md [hunk 3]
```diff
@@ -1,5 +1,61 @@
...
+
+ ## Types of Events
+ 
+ Identifying whether your workshop will be a self-paced workshop or an AWS hosted event will be crucial in your planning as well. If it’s a self-paced workshop then having highly contextualized sections will be very important as there won’t be anyone there to answer questions. Making sure that the workshop itself has all relevant information or that clear references have been outlined for any documentation that will be needed to successfully complete the workshop are crucial to the workshops overall success.
```
Comment (eugenemu) [inline README.md:29]: For the mainline branch, we need the setup instruction to just have Event Engine setup. There's a separate marketplace branch where we can add the self-hosted setup.

File: README.md [hunk 4]
```diff
@@ -1,5 +1,61 @@
...
+Contributors names and contact info
+
+* James Bland (@jamesbland123)
+* Parker Perry (@parkerperry)  
+* James Spencer (@folrig)
+* Eugene Mu (@eugenemu)
+
+## License
+
+This project is licensed. See the LICENSE.md file for details
```
Comment (eugenemu) [inline README.md:54]: I don't think any of the MADs are licensed, we may need to remove this line and confirm.

File: config.toml [hunk 5]
```diff
@@ -1,6 +1,6 @@
 baseURL = "/"
 languageCode = "en-US"
-defaultContentLanguage = "en"
+defaultContentLanguage = "selfguided" # Modify to change the default language/version of the site
```
Comment (eugenemu) [inline config.toml]: Revert this change, as we're only using the defaultContentLanguage as "en"

File: config.toml [hunk 6]
```diff
@@ -30,7 +30,7 @@ enableEmoji = true
   # A title for shortcuts in menu is set by default. Set this to true to disable it. 
   disableShortcutsTitle = false
   # When using mulitlingual website, disable the switch language button.
-  disableLanguageSwitchingButton = true
+  disableLanguageSwitchingButton = false
```
Comment (eugenemu) [inline config.toml]: Revert this change.

File: config.toml [hunk 7]
```diff
@@ -77,4 +77,15 @@ weight = 23
 [markup]
   defaultMarkdownHandler = "blackfriday"
   [markup.blackFriday]
-    hrefTargetBlank = true
\ No newline at end of file
+    hrefTargetBlank = true
```
Comment (eugenemu) [inline config.toml:80]: The above changes in config.toml will be needed in the mp-workshop branch.

File: content/1_ModuleOne/11_Foreward.ee.md [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+chapter: true
+weight: 1 # MODIFY THIS VALUE TO REFLECT THE ORDERING OF THE MODULES IF APPLICABLE
+---
+
+# Foreward <!-- MODIFY THIS HEADING IF APPLICABLE -->
```
Comment (eugenemu) [inline content/1_ModuleOne/11_Foreward.ee.md]: Foreword

File: content/1_ModuleOne/11_Foreward.md [hunk 9]
```diff
@@ -0,0 +1,23 @@
+---
+title: "Foreward" # MODIFY THIS TITLE IF APPLICABLE
```
Comment (eugenemu) [inline content/1_ModuleOne/11_Foreward.md]: Foreword

File: content/1_ModuleOne/11_Foreward.md [hunk 10]
```diff
@@ -0,0 +1,23 @@
...
+chapter: true
+weight: 1 # MODIFY THIS VALUE TO REFLECT THE ORDERING OF THE MODULES IF APPLICABLE
+---
+
+# Foreward <!-- MODIFY THIS HEADING IF APPLICABLE -->
```
Comment (eugenemu) [inline content/1_ModuleOne/11_Foreward.md]: Foreword

File: content/1_ModuleOne/12_PartnerIdentification.ee.md [hunk 11]
```diff
@@ -0,0 +1,23 @@
+---
+title: "Partner Identification" # REPLACE THIS TITLE WITH THE PARTNER NAME
+chapter: true
+weight: 3 # MODIFY THIS VALUE TO REFLECT THE ORDERING OF THE MODULES IF APPLICABLE
+---
+
+# Partner Identification <!-- MODIFY THIS HEADING WITH THE PARTNER NAME -->
+
+## Submodule Three Heading <!-- MODIFY THIS SUBHEADING -->
```
Comment (eugenemu) [inline content/1_ModuleOne/12_PartnerIdentification.ee.md]: We many not need this section - up to the partner on how they want to market themselves.

File: content/1_ModuleOne/13_TechnicalIssue_Problem.ee.md [hunk 12]
```diff
@@ -0,0 +1,23 @@
+---
+title: "Technical Issue / Problem" # MODIFY THIS TITLE
+chapter: true
+weight: 2 # MODIFY THIS VALUE TO REFLECT THE ORDERING OF THE MODULES IF APPLICABLE
```
Comment (eugenemu) [inline content/1_ModuleOne/13_TechnicalIssue_Problem.ee.md]: We should also provide guidance on naming the files, i.e. when you add 13_TechnicalIssue_Problem.md but the weight is 2, it's a bit contradictory since yes, it will show in the correct order by weight, but from a file standpoint, this file will show up after the partner identification file.

File: content/1_ModuleOne/14_WorkshopPrerequisites.md [hunk 13]
```diff
@@ -0,0 +1,23 @@
+---
+title: "Workshop Prerequisites" # MODIFY THIS TITLE IF APPLICABLE
+chapter: true
+weight: 4 # MODIFY THIS VALUE TO REFLECT THE ORDERING OF THE MODULES IF APPLICABLE
+---
+
+# Workshop Prerequisites <!-- MODIFY THIS HEADING IF APPLICABLE -->
+
+## Submodule Four Heading <!-- MODIFY THIS SUBHEADING -->
```
Comment (eugenemu) [inline content/1_ModuleOne/14_WorkshopPrerequisites.md:9]: We can be a bit more prescriptive here, instead of saying submodule headings, add what should be added to this page: i.e. "What are the prerequisite tasks the workshop attendees must have done in order to do the workshop", here's an example: https://harness.awsworkshop.io/010_introduction/16_workshop_next_step.html

File: content/1_ModuleOne/13_TechnicalIssue_Problem.md [hunk 14]
```diff
@@ -0,0 +1,23 @@
+---
+title: "Technical Issue / Problem" # MODIFY THIS TITLE
+chapter: true
+weight: 2 # MODIFY THIS VALUE TO REFLECT THE ORDERING OF THE MODULES IF APPLICABLE
+---
+
+# Technical Issue / Problem <!-- MODIFY THIS HEADING TO REFLECT THE PROBLEM THE WORKSHOP IS ADDRESSING -->
+
+## Submodule Two Heading <!-- MODIFY THIS SUBHEADING -->
```
Comment (eugenemu) [inline content/1_ModuleOne/13_TechnicalIssue_Problem.md]: Instead of markdown guidance, add more content guidance. Example: https://harness.awsworkshop.io/010_introduction/12_problem_to_solve.html

File: content/1_ModuleOne/12_PartnerIdentification.md [hunk 15]
```diff
@@ -0,0 +1,23 @@
+---
+title: "Partner Identification" # REPLACE THIS TITLE WITH THE PARTNER NAME
+chapter: true
+weight: 3 # MODIFY THIS VALUE TO REFLECT THE ORDERING OF THE MODULES IF APPLICABLE
+---
+
+# Partner Identification <!-- MODIFY THIS HEADING WITH THE PARTNER NAME -->
+
+## Submodule Three Heading <!-- MODIFY THIS SUBHEADING -->
```
Comment (eugenemu) [inline content/1_ModuleOne/12_PartnerIdentification.md]: Add content guidance of "Describe partner company, what is the marketing language the partner wants to use? What are the platform highlights, use cases it solves, success metrics etc."

File: content/1_ModuleOne/15_NextSteps.ee.md [hunk 16]
```diff
@@ -0,0 +1,23 @@
+---
+title: "Next Steps" # MODIFY THIS TITLE TO REFLECT NEXT STEPS IF APPLICABLE
+chapter: true
+weight: 5 # MODIFY THIS VALUE TO REFLECT THE ORDERING OF THE MODULES IF APPLICABLE
+---
+
+# Next Steps <!-- MODIFY THIS HEADING IF APPLICABLE -->
+
+## Submodule Five Heading <!-- MODIFY THIS SUBHEADING -->
```
Comment (eugenemu) [inline content/1_ModuleOne/15_NextSteps.ee.md:9]: More content guidance - is there anything the partner needs to do as a next step? This page is typically optional, but the partner might need it.

File: content/1_ModuleOne/_index.ee.md [hunk 17]
```diff
@@ -0,0 +1,38 @@
...
+# Introduction
+
+## Module One Heading <!-- MODIFY THIS SUBHEADING -->
+
+This paragraph block should be an introduction to the module.
+
+### Module One Subheading <!-- MODIFY THIS SUBHEADING -->
+This paragraph block should be utilized to briefly explain the submodules that are going to be presented. <br>
+
+#### Foreward <!-- MODIFY THIS SUBHEADING -->
```
Comment (eugenemu) [inline content/1_ModuleOne/_index.ee.md]: Foreword

File: content/1_ModuleOne/_index.ee.md [hunk 18]
```diff
@@ -0,0 +1,38 @@
+---
+title: "Introduction"
+chapter: true
+weight: 1
+---
+
+# Introduction
+
+## Module One Heading <!-- MODIFY THIS SUBHEADING -->
```
Comment (eugenemu) [inline content/1_ModuleOne/_index.ee.md]: Follow conventions here: https://circleci.awsworkshop.io/

There should be the partner logo at the top of the page, there should be the warning label on all intro pages, and the welcome message should have description of what needs to go in there, i.e. 2-3 sentences on what the workshop is going to be about, the use case it's solving either generic or otherwise.

File: content/2_ModuleTwo_Setup/21_SelfGuidedSetup.md [hunk 19]
```diff
@@ -0,0 +1,26 @@
+---
+title: "Self-Guided Setup Instructions" # MODIFY THIS TITLE
+chapter: true
+weight: 1 # MODIFY THIS VALUE TO REFLECT THE ORDERING OF THE MODULES
+---
+
+<!-- MORE SUBMODULES CAN BE ADDED TO DIVIDE UP THE SETUP INTO SMALLER SECTIONS -->
+<!-- COPY AND PASTE THIS SUBMODULE FILE, RENAME, AND CHANGE THE CONTENTS AS NECESSARY -->
+
```
Comment (eugenemu) [inline content/2_ModuleTwo_Setup/21_SelfGuidedSetup.md]: More content guidance ala: https://circleci.awsworkshop.io/030_self_guided_setup/30_aws_setup_your_own.html, with the exception of the warning note as it's only relevant to MP workshop series.

Most of the content should have these instructions verbatim.

File: content/2_ModuleTwo_Setup/_index.ee.md [hunk 20]
```diff
@@ -0,0 +1,35 @@
...
+title: "AWS Hosted Workshop Setup" # MODIFY THIS TITLE IF APPLICABLE
+chapter: true # MAKE SURE YOU SET THIS TO FALSE IF USING SELF-GUIDED SETUP
+weight: 2
+---
+
+# AWS Hosted Workshop Setup <!-- MODIFY THIS HEADING -->
+
+## Module Two Heading <!-- MODIFY THIS SUBHEADING -->
+
+This paragraph block should be an introduction to the module about the method of setting up the workshop using EventEngine in AWS. Some typical examples of the steps in this process will explain logging out of any current AWS accounts, how to utilize the hashed URL, utilizing the Team Dashboard to log in to the AWS console, and ensure that the correct region is being selected.
```
Comment (eugenemu) [inline content/2_ModuleTwo_Setup/_index.ee.md:11]: Content guidance can be viewed here: https://circleci.awsworkshop.io/ee/020_event_engine_setup.html

File: content/3_ModuleThree/31_PartnerSetup.ee.md [hunk 21]
```diff
@@ -0,0 +1,23 @@
+---
+title: "Partner Setup Instructions" # MODIFY THIS TITLE
+chapter: true
+weight: 1 # MODIFY THIS VALUE TO REFLECT THE ORDERING OF THE MODULES
+---
+
+# Partner Setup Instructions <!-- MODIFY THIS HEADING -->
```
Comment (eugenemu) [inline content/3_ModuleThree/31_PartnerSetup.ee.md:7]: More content guidance.

Comment (eugenemu) [reply content/3_ModuleThree/31_PartnerSetup.ee.md:7]: i.e. does the partner require attendees to sign up for things?
https://circleci.awsworkshop.io/ee/040_circleci_setup.html
a la signing up for Docker Hub, Snyk, Terraform etc.

**Review by eugenemu — CHANGES_REQUESTED** [review]
Overall, we need more content guidance on some of the pages: what should each page/subpage mean and what content should the partner add there?

File: content/1_ModuleOne/11_Foreward.ee.md (hunk 1 above)
Comment (Folrig) [reply content/1_ModuleOne/11_Foreward.ee.md]: Spoke with @eugenemu, going to leave this in for now with explicit guidance to remove the Event Engine language from the config.toml if it is not applicable to the workshop

File: content/1_ModuleOne/11_Foreward.ee.md (hunk 2 above)
Comment (Folrig) [reply content/1_ModuleOne/11_Foreward.ee.md]: Fixed

File: content/1_ModuleOne/_index.ee.md (hunk 17 above)
Comment (Folrig) [reply content/1_ModuleOne/_index.ee.md]: Fixed

File: content/1_ModuleOne/11_Foreward.ee.md (hunk 8 above)
Comment (Folrig) [reply content/1_ModuleOne/11_Foreward.ee.md]: Fixed

File: content/1_ModuleOne/11_Foreward.md (hunk 9 above)
Comment (Folrig) [reply content/1_ModuleOne/11_Foreward.md]: Fixed

File: content/1_ModuleOne/15_NextSteps.ee.md (hunk 16 above)
Comment (ParkerPerry) [reply content/1_ModuleOne/15_NextSteps.ee.md:9]: Not exactly sure what to add here. Will come back to and revisit

File: README.md (hunk 4 above)
Comment (Folrig) [reply README.md:54]: The baseline repo is licensed under the LICENSE.md. This can be removed later if necessary

File: content/1_ModuleOne/12_PartnerIdentification.md (hunk 15 above)
Comment (Folrig) [reply content/1_ModuleOne/12_PartnerIdentification.md]: Removed as per previous issue. Can be added back as necessary

File: README.md (hunk 3 above)
Comment (Folrig) [reply README.md:29]: Will need clarification. Previous issues say to remove .ee files

**Review by eugenemu — APPROVED** [review]
